common/main/af.xml

```diff
@@ -6160,7 +6160,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			</compoundUnit>
 			<unit type="acceleration-g-force">
 				<displayName>swaartekrag</displayName>
-				<unitPattern count="one">{0} swaartekrag</unitPattern>
+				<unitPattern count="one">↑↑↑</unitPattern>
```

Comment:
what makes this clearly wrong is it's replacing a hard value with an inheritance marker

and in a path that didn't allow changes

Reply:
"clearly" being subjective and not the adverb I would use - the above note is from what was said in the meeting